src/util/runExtension.ts

```diff
@@ -338,16 +338,6 @@ Please remove the \`{{[[roam/js]]}}\` code that installed this extension and ref
             },
           },
         },
-        ui: {
-          commandPalette: {
-            addCommand: ({ label, callback, disableHotkey, defaultHotkey }) => {
-                  label
-                  callback
-                  disableHotkey
-                  defaultHotkey
-                },
-            },
-        },
```

Comment:
Does this not fail type checking? Ideally we might be able to delete this whole thing, which is something I might do after this PR is merged

Reply:
`extensionAPI.ui` was made optional.